Fyber FairBid/AdsScreenViewController.swift

```diff
@@ -235,8 +235,8 @@ extension AdsScreenViewController: FYBRewardedDelegate {
     func rewardedDidShow(_ placementName: String, impressionData: FYBImpressionData) {
         addEventToCallbacksList(#function)
     }
-
-    func rewardedDidFail(toShow placementName: String, withError error: Error) {
+    
+    func rewardedDidFail(toShow placementId: String, withError error: Error, impressionData: FYBImpressionData) {
```

Comment:
I would make the `placementName` -> `placementId` change everywhere

Reply:
I refactored them in this commit 
c385997